src/TaskManager/Plug-ins/AideClinicalReview/AideClinicalReviewPlugin.cs

```diff
@@ -114,6 +123,36 @@ private void Initialize()
             {
                 _reviewedTaskId = Event.TaskPluginArguments[Keys.ReviewedTaskId];
             }
+
+            if (Event.TaskPluginArguments.ContainsKey(Keys.ApplicationName))
+            {
+                _applicationName = Event.TaskPluginArguments[Keys.ApplicationName];
+            }
+
+            if (Event.TaskPluginArguments.ContainsKey(Keys.ApplicationVersion))
+            {
+                _applicationVersion = Event.TaskPluginArguments[Keys.ApplicationVersion];
+            }
+
+            if (Event.TaskPluginArguments.ContainsKey(Keys.Mode))
+            {
+                _mode = Event.TaskPluginArguments[Keys.Mode];
+            }
+
+            if (Event.TaskPluginArguments.ContainsKey(Keys.ReviewerRoles))
```

Comment:
What if the TaskPluginArguments does not contain ReviewerRoles? Should we default to "clinician" as well?

Reply:
Good spot, fixed to default if missing.